requirements.txt

```diff
@@ -12,7 +12,7 @@ freezegun
 pyreact
 Flask-Caching
 blinker
-pycrypto
+pycryptodome
```

Comment:
What is the difference here?

Reply:
`pycrypto` is quite an old thing, and it refuse to compile on my Windows machine. Therefore I've switched to that forked version.